newIDE/app/src/Utils/ProjectCache.js

```diff
@@ -17,12 +17,33 @@ class ProjectCache {
 
   static async burst() {
     if (!ProjectCache.isAvailable()) return;
+    const databases = await window.indexedDB.databases();
+    if (
+      !databases.find(
+        database => database.name === CLOUD_PROJECT_AUTOSAVE_CACHE_KEY
+      )
+    ) {
+      // The database does not exist so there is nothing to clear.
+      return;
+    }
     return new Promise(resolve => {
       const request = window.indexedDB.open(CLOUD_PROJECT_AUTOSAVE_CACHE_KEY);
       request.onsuccess = event => {
         const db = event.target.result;
-        const transaction = db.transaction(objectStoreScope, 'readwrite');
-        transaction.objectStore(objectStoreScope).clear();
+        if (!db.objectStoreNames.contains(objectStoreScope)) {
+          // The database does not contain the object store so there is nothing to clear.
+          // This situation will be fixed by the next database initialization.
+          resolve();
+        }
+        try {
+          const transaction = db.transaction(objectStoreScope, 'readwrite');
+          transaction.objectStore(objectStoreScope).clear();
+        } catch (error) {
+          console.warn(
```

Comment:
Because of the previous `if (!db.objectStoreNames.contains(objectStoreScope))`, this should never happen, but the error is still caught just in case right?

Reply:
Yes, I will add a comment